internal/testrunner/runners/pipeline/ingest_pipeline.go

```diff
@@ -0,0 +1,219 @@
+package pipeline
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/elastic/go-elasticsearch/v7/esapi"
+	"io/ioutil"
+	"log"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/elastic/go-elasticsearch/v7"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
+
+	"github.com/elastic/elastic-package/internal/packages"
+)
+
+const defaultPipelineName = "default"
+
+var ingestPipelineTag = regexp.MustCompile("{{ IngestPipeline.+ }}")
```

Comment:
Probably want to make this a bit more forgiving?

```suggestion
var ingestPipelineTag = regexp.MustCompile("{{\s*IngestPipeline.+}}")
```

Reply:
Agree, fixed.